src/current/v23.1/changefeed-messages.md

```diff
@@ -248,6 +248,41 @@ There are three different ways to configure resolved timestamp messages:
 If you require `resolved` message frequency under `30s`, then you **must** set the [`min_checkpoint_frequency`](create-changefeed.html#min-checkpoint-frequency) option to at least the desired `resolved` frequency. This is because `resolved` messages will not be emitted more frequently than `min_checkpoint_frequency`, but may be emitted less frequently.
 {{site.data.alerts.end}}
 
+## Duplicate messages
+
+Under some circumstances, changefeeds will emit duplicate messages to ensure the sink is receiving each message at least once. The following can cause duplicate messages:
+
+- The changefeed job [encounters an error](#changefeed-encounters-an-error).
+- A node in the cluster [restarts](#node-restarts).
+- The changefeed job has the [`min_checkpoint_frequency` option set](#min_checkpoint_frequency-option).
+- A target table undergoes a schema change. Refer to [Schema changes](#schema-changes) for detail on duplicates in this case.
```

Comment:
Up to you, but it might be worth emphasizing here (for those who might make assumptions and not follow the link), that this can include emitting the entire table, not just recent messages.

Reply:
Yes, we should do that. I have included an additional sentence to that effect.